packages/frontend/package.json

```diff
@@ -29,10 +30,11 @@
     "@testing-library/user-event": "^14.5.1",
     "@tsconfig/svelte": "^5.0.2",
     "@typescript-eslint/eslint-plugin": "6.15.0",
+    "jsdom": "^23.2.0",
     "svelte": "4.2.8",
     "svelte-preprocess": "^5.1.3",
     "tailwindcss": "^3.4.0",
     "postcss": "^8.4.33",
-    "postcss-load-config": "^5.0.2"
+    "postcss-load-config": "^5.0.2"    
```

Comment:
it looks like the file is not formatted/sorted (usually it is ordered in alphabetical order) and there is an extra space here

Reply:
how do you sort it automatically? Is there any npm package? 🤔

Reply:
I think it is done when you run `yarn add <package>`

Reply:
I was hoping we could add some script so that if someone copy/paste an entry it is sorted anyway

Reply:
`npx sort-package-json`

Reply:
(anyway it's not a requested change)

Reply:
cool, i'm going to add it in a follow-up PR so i don't block @feloy 
for the moment i fixed it manually